src/escape_chars.c

```diff
@@ -29,6 +29,9 @@ SEXP C_escape_chars_one(SEXP x) {
       case '\b':
       case '\f':
         matches++;
+        break;
+      case '/':
+        matches += escape_solidus;
```

Comment:
Is there a reason not to add this to the cases above, and simply use `matches++`?